scripts/create_admin_account.py

```diff
@@ -10,12 +9,15 @@
 
 import bcrypt
 
+
+docker_fp = "/app/config/default.json"
```

Comment:
Same as other comments, this should be redone either using `__file__` or a command line argument to get the path.